rdfloader/parser2v2/utils_test.go

```diff
@@ -184,106 +151,3 @@ func Test_getNodeTypeFromTriples(t *testing.T) {
 		t.Errorf("expected an error saying more than one rdf:type found, got %v", err)
 	}
 }
-
-// following tests are copy pasted from tvloader/parser2v2/util_test.go
-
-func TestCanExtractDocumentAndElementRefsFromID(t *testing.T) {
```

Comment:
I made sure to hang on to non-duplicate tests and move them to the proper place after my re-arranging, so we did not lose any coverage.